python/pyarrow/types.pxi

```diff
@@ -1124,7 +1124,7 @@ cdef class Time32Type(DataType):
     @property
     def unit(self):
         """
-        The time unit ('s', 'ms', 'us' or 'ns').
+        The time unit ('s' or 'ms').
```

Comment:
Can you maybe also mention the supported units in the docstring of the class just above? (same for Time64)

Reply:
Good point, will do!